input/postgres/replication.go

```diff
@@ -47,9 +59,30 @@ func GetReplication(ctx context.Context, logger *util.Logger, db *sql.DB, postgr
 	var err error
 	var repl state.PostgresReplication
 	var sourceTable string
+	var replicationSQL string
 
 	if postgresVersion.IsAwsAurora {
-		// Most replication functions are not supported on AWS Aurora Postgres
+		if !auroraDbInstanceIdentifierExists(ctx, db) {
```

Comment:
what exactly the case/occasion that IsAwsAurora is true but aurora DB instance Identifier doesn't exist?

Reply:
(also the comment of `getIsReplicaAurora` function is not so clear to me; does `auroraDbInstanceIdentifierExists` being false mean that it's very old Aurora version...??)

Reply:
That's right, if `auroraDbInstanceIdentifierExists` is false its a very old Aurora release, and in such situations we don't have an easy way to identify the current instance in the list of replicas. Whilst we could pull the instance ID from the server config, this wasn't super straightforward, and I think its okay to essentially do what we used to do in these cases.